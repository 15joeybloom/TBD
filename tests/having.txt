select gradyear, count(sid) from student group by gradyear having count(sid) = 2
