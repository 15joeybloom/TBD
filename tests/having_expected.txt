 count(sid) gradyear
--------------------
          2     2005
          2     2001
